workspaces/bulk-import/plugins/bulk-import-backend/src/gitlab/gitlabApiService.ts

```diff
@@ -322,19 +315,19 @@ export class GitlabApiService {
       },
       this.integrations,
       {
-        dataFetcher: async (gitlab: any) => {
+        dataFetcher: async (gitlab: InstanceType<typeof Gitlab<false>>) => {
           // find authenticated gitlab owner...
           const username = (await gitlab.Users.showCurrentUser()).username;
           if (username) {
             allAccessibleUsernames.add(username);
           }
           // ... along with orgs accessible from the token auth
           (
-            await gitlab.Groups.all({
-              all_available: false,
+            await gitlab.Groups.all<true, 'offset'>({
+              allAvailable: false,
             })
-          )
-            ?.map((org: { path: any }) => org.path)
+          )?.data
```

Comment:
There is actually no "data" attribute that is returned when the api is called like that `gitlab.Groups.all`,  it actually just returns an array of JSON values.
```
[
  {...},
  {...}
]
```

so that is why the test is failing.

Reply:
You would need to add the showExpanded param to get that data attribute in the result, however, this code still wouldn't work

```
console.log(await api.Groups.all({allAvailable: false, showExpanded: true})?data);
```

will still return undefined, the only way to access it is using some desctructuring like this:

```
const {data} = await api.Groups.all({allAvailable: false, showExpanded: true});
```